### PR TITLE
Admin navbar links

### PR DESCRIPTION
**Pull request by Lydias303**
The admin navbar links are now functional! I hooked up all the links with the routes that Jeff had made. 

The pages namespaced under the static controller required controller actions and views to be made so I went ahead and did that. 

I also updated all the tests to reflect the changes and added new ones for all of the view as user links.

I also added buttons on the admin dashboard to link to an edit admin account and add admin. Neither are hooked up yet.  I plan to work with Krista to implement the background functionality for that next step.

### CODE REVIEW

File: app/controllers/admin/static_pages/static_pages_controller.rb [hunk 1]
```diff
@@ -1,5 +1,21 @@
...
+
   def learn_more
+    @pdf = Pdf.learn_more
+    @contents = Content.where(page: "learn more")
+  end
+
+  def techhire_locations
+    @techhire_locations = TechhireLocation.all
+  end
+
```
Comment (Jwan622) [inline app/controllers/admin/static_pages/static_pages_controller.rb:11]: dope. I like these pages and what we have planned for the CMS.

File: app/views/admin/dashboard/index.html.erb [hunk 2]
```diff
@@ -1,29 +1,52 @@
...
-  </div>
-  <div class="companies-by-city col-md-4">
-    <h3>Top TechHire Cities</h3>
-    <% Company.company_count_by_city.each do |city, count|%>
-    <div class="top-cities">
-      <%= city %> : <%= count %>
-    </div>
-    <% end %>
+  <div class="col-md-8">
+    <%= render 'uncontacted_cities' %>
```
Comment (Jwan622) [inline app/views/admin/dashboard/index.html.erb:50]: pretty move.

File: app/views/application/_navbar.html.erb [hunk 3]
```diff
@@ -14,31 +14,40 @@
...
+                    <li class="dropdown-header manage-learn"><a href="/admin/static_pages/learn_more">Learn More <span class="glyphicon glyphicon-education"></span></a></li>
+                    <li class="dropdown-header manage-home"><a href="/admin/static_pages/home">Home <span class="glyphicon glyphicon-home"></span></a></li>
                   </ul>
                     </li>
                   </ul>
                     </li>
                   </ul>
-                  <a class="nav navbar-right" href="#">Logout</a>
+                  <div class="nav navbar-right">
+                  <%= link_to "Logout", destroy_user_session_path(current_user.id), method: :delete %>
```
Comment (Jwan622) [inline app/views/application/_navbar.html.erb:49]: Love it, great job on this nav bar.

File: app/views/static_pages/home.html.erb [hunk 4]
```diff
@@ -3,6 +3,7 @@
 <div class="links col-md-12">
   <%= link_to "Learn More", learn_more_path %>
   <%= link_to "Sign Up", signup_path %>
+  <%= link_to "Sign In", new_user_session_path %>
```
Comment (Jwan622) [inline app/views/static_pages/home.html.erb:6]: This makes life easier. Thanks.

**Comment by bhargavisatpathy** [comment]
Looks good to me to merge :)

File: spec/features/admin_dashboard_spec.rb [hunk 5]
```diff
@@ -126,18 +138,71 @@ def login
...
+    end
+
+    it "can click a link to view the techhire locations page" do
+      login
+
+      expect(current_path).to eq(admin_dashboard_index_path)
+
+      within("li.view-techhire-locations") do
+        expect(page).to have_content('Tech Hire Locations')
+        click_link_or_button('Tech Hire Locations')
```
Comment (Jwan622) [inline spec/features/admin_dashboard_spec.rb:203]: great tests for integration. I like the idea that you click links and you verify path.

File: app/views/admin/dashboard/index.html.erb [hunk 6]
```diff
@@ -1,29 +1,52 @@
+<div class="container">
+  <div class="panel-body col-md-4">
+    <ul class="nav navbar-nav">
+      <li class="panel panel-info" id="dropdown">
+        <a data-toggle="collapse" href="#dropdown-lvl1">
+          <h3>Manage Administrators</h3><span class="caret"></span>
+        </a>
+        <div id="dropdown-lvl1" class="panel-collapse collapse">
```
Comment (scottalexandra) [inline app/views/admin/dashboard/index.html.erb:8]: I like this addition, its really clean!

**Pull request merged**